packages/mds-ingest-service/@types/index.ts

```diff
@@ -109,6 +109,14 @@ export type GetVehicleEventsResponse = {
   }
 }
 
+export interface ReadTripEventsQueryParams {
+  skip?: UUID
```

Comment:
Decided on skip/take over limit/cursor here?

Reply:
Yeah, this was a direct translation of the original mds-db query we wrote months ago, so I didn't have any reasons to refactor the interface or behavior in isolation of changes to the trip-log report processor.